### ISSUE TITLE
Vector overlays are not rendering for Mac retina

### ISSUE THREAD

**Issue by nmwolf** [NYULibraries/spatial_data_repository#308]
A setting configuration for the platform to account for Mac retina screens was uncovered during sprint, but it seems not to be working. Mac retina screens show fuzzy vector polygon boundaries.

![Image](https://github.com/NYULibraries/spatial_data_repository/assets/11075283/eaf8652f-b0cf-4f3a-865b-cfede6f34c34)

**Comment by spilth** [comment]
@nmwolf Do you know which record is shown above?

**Comment by nmwolf** [comment]
Yes, it is [https://geo.nyu.edu/catalog/nyu-2451-34630](https://geo.nyu.edu/catalog/nyu-2451-34630).
Confirmed on my retina at home that the quality is still fuzzy.

Here's another example just to show it isn't a single record, from [https://geo.nyu.edu/catalog/nyu-2451-34445](https://geo.nyu.edu/catalog/nyu-2451-34445)
<img width="874" alt="Screenshot 2024-11-25 at 8 59 05 PM" src="https://github.com/user-attachments/assets/0c71c69b-0184-4f3d-9a19-e5abeb8d2ef8">
